src/topology/locally_constant/basic.lean

```diff
@@ -222,8 +223,24 @@ coe_injective (funext h)
 theorem ext_iff {f g : locally_constant X Y} : f = g ↔ ∀ x, f x = g x :=
 ⟨λ h x, h ▸ rfl, λ h, ext h⟩
 
-protected lemma continuous [topological_space Y] (f : locally_constant X Y) : continuous f :=
-f.is_locally_constant.continuous
+section codomain_topological_space
+
+variables [topological_space Y] (f : locally_constant X Y)
+
+protected lemma continuous : continuous f := f.is_locally_constant.continuous
+
+/-- We can turn a locally-constant function into a bundled `continuous_map`. -/
+def to_continuous_map : C(X, Y) := ⟨f, f.continuous⟩
+
+instance : has_coe (locally_constant X Y) C(X, Y) := ⟨to_continuous_map⟩
+
+@[simp] lemma to_continuous_map_coe : ((f : C(X, Y)) : X → Y) = (f : X → Y) := rfl
+
+lemma to_continuous_map_injective {f g : locally_constant X Y}
+  (h : (f : C(X, Y)) = (g : C(X, Y))) : f = g :=
```

Comment:
Is there a reason you didn't state this with `function.injective`?

Reply:
Nope, now done in https://github.com/leanprover-community/mathlib/pull/8444/commits/5f03853339b775360e0863996fe7de9c0c07c3b4